setup.cfg

```diff
@@ -1,8 +1,12 @@
 [metadata]
-name = matrix-python-common
+name = matrix_common
```

Comment:
Is this change wanted?

Reply:
I think the intent is to make the module/package name `matrix_common` the same as the thing on PyPI?

Reply:
It's neither intentional nor unintentional. I did notice the change but didn't feel too strongly about it.

It brings `setup.cfg` closer in line with the template. It does have the effect of changing the PyPI package name from `matrix-python-common` to `matrix-common`. I'm open to keeping it as `matrix-python-common`.

I think this is distinct from the `import` name / module directory name, which is already `matrix_common`. Or would be if I could figure out why the packages I build are all empty.